### DISCUSSION TITLE
Help required on GraphQL stitching

### DISCUSSION THREAD

**Discussion by amityweb** [vercel/next.js#40098] [Help]
I have two GraphQLs that I need to merge into one, so we can run queries that return data from both sets. e.g. Saleor eCommerce system and Strapi CMS. Saleor will have product IDs for checkout and stock, Strapi has all product content. 

I am having a real hard time finding any help. The very few guides online don't work, errors I can resolve or I don't understand them. 

Does anyone have any simple guide on how to merge two GraphQLs? I have two GraphQL urls. In the Strapi data is a field SaleorID which contains the ID of data from the Saleor data. Coming from a PHP/MySQL background I assume such querying would be easy but I have probably spent 1 to 2 weeks worth of hours with no progress. 

Thanks

**Comment by paales** [comment]
We're doing that with [GraphCommerce](https://github.com/graphcommerce-org/graphcommerce), in out case merging GraphCMS and the Magento 2 GraphQL API with the help of [GraphQL Mesh](https://www.graphql-mesh.com/). Example configuration https://github.com/graphcommerce-org/graphcommerce/blob/main/examples/magento-graphcms/.meshrc.yml

**Comment by amityweb** [comment]
Thanks. I have finally figured it out, almost, using an older GitHub repo tutorial (the new one replaced the method in a way I did not want to do): https://github.com/royderks/saleor-strapi-stitching/tree/e702434c2d13de629b62f45f3fe1f54892e44e79

Although I cant get the second schema data in into the first yet! I am sure its 99% there!
